exercises/prime-factors/README.md

```diff
@@ -24,26 +24,24 @@ factors of 60: 2, 2, 3, and 5.
 
 You can check this yourself:
 
-- 2 * 2 * 3 * 5
-- = 4 * 15
+- 2 _ 2 _ 3 \* 5
```

Comment:
```suggestion
- 2 \* 2 \* 3 \* 5
```

I don't know if escaping `*`s is actually necessary with Exercism's style of Markdown.